R/report_results.R

```diff
@@ -403,7 +408,7 @@ report_results<-function(kmer_hist,kmer_hist_orig, k, p, container, foldername,
     dev.set(dev.next())
 
     ## Finish Linear Plot
-    title(paste("\n\nlen:",  prettyNum(total_len[1], big.mark=","),
+    title(paste("\n\nlen:",  prettyNum(atotal_len, big.mark=","),
```

Comment:
This section is a bit confusing to me - while other fitted variables (`kcov`, `dup`) are reported as averages, the genome length was for a reason I don't understand reported as the minimal estimate.

I changed it to average too, but as I don't understand the reasoning behind reporting minimum, I might be missing something.

Reply:
I think that was just legacy from what GS1 reported. But agree it would make more sense to report the average